package.json

```diff
@@ -86,7 +87,7 @@
     "marked": "^0.7.0",
     "pluralize": "^8.0.0",
     "prop-types": "^15.7.2",
-    "query-string": "^6.8.3",
+    "query-string": "^4.1.0",
```

Comment:
I did not experiment with any query-string 5.X.X because at this point, I am extremely risk-avert. I may do so in a future npm update.